services/cd-service/Buildfile.yaml

```
@@ -9,3 +9,5 @@ spec:
   buildWith: infrastructure/docker/builder
   dependsOn:
   - ../../
+additional_artifacts:
+  path: coverage.out
```

You don't need to add `path:`, `addition_artifacts` expects a list of files.
```suggestion
  - coverage.out
```